docs/learn/documentation/versioned/jobs/samza-configurations.md

```diff
@@ -321,7 +321,8 @@ These properties define Samza's storage mechanism for efficient [stateful stream
 |stores.**_store-name_**.<br>rocksdb.keep.log.file.num|2|The number of RocksDB LOG files (including rotated LOG.old.* files) to keep.|
 |stores.**_store-name_**.<br>rocksdb.metrics.list|(none)|A list of [RocksDB properties](https://github.com/facebook/rocksdb/blob/master/include/rocksdb/db.h#L409) to expose as metrics (gauges).|
 |stores.**_store-name_**.<br>rocksdb.delete.obsolete.files.period.micros|21600000000|This property specifies the period in microseconds to delete obsolete files regardless of files removed during compaction. Allowed range is up to 9223372036854775807.|
-|stores.**_store-name_**.<br>rocksdb.max.manifest.file.size|18446744073709551615|This property specifies the maximum size of the MANIFEST data file, after which it is rotated. Default value is also the maximum, making it practically unlimited: only one manifest file is used.|
```

Comment:
The number was outdated, and it should be 1024^3 which is 1073741824 
- The RocksDB version for Samza was upgraded from `5.7.3` to `6.5.3` at https://github.com/apache/samza/commit/0189b5dc161501c572220de0a932180b6649a1e1
- In `5.7.3`, the `max_manifest_file_size` defaults to uint64 max which is `18446744073709551615` (https://github.com/facebook/rocksdb/blob/v5.7.3/include/rocksdb/options.h#L547)
- In `6.5.3`, the `max_manifest_file_size` defaults to 1GB which is `1024*1024*1024` (https://github.com/facebook/rocksdb/blob/v6.5.3/include/rocksdb/options.h#L623)